### ISSUE TITLE
Specify Android SDK version number in README.md

### ISSUE THREAD

**Issue by filipedelimabrito** [jitsi/jitsi-meet#3987]
It's a good practice to avoid using `+` in version number as we can see on the Android [README.md](https://github.com/jitsi/jitsi-meet/blob/master/android/README.md#use-pre-build-sdk-artifactsbinaries) file: 

```
dependencies {
    // (other dependencies)
    implementation ('org.jitsi.react:jitsi-meet-sdk:+') { transitive = true }
}
```

Is there any possibility to include the most recent version number instead of instructing people doing that?

**Comment by saghul** [comment]
Good point. We actually have a 2.0 version of it in master which will be released soon. I'll make sure we update the README. Thanks!

**Comment by filipedelimabrito** [comment]
@saghul I could see a 2.0.0 release here, but the [README.md](https://github.com/jitsi/jitsi-meet/blob/master/android/README.md#use-pre-build-sdk-artifactsbinaries) is still recommending us to use  `+` there. When this issue can be closed?

**Comment by saghul** [comment]
I consider the README now to be correct. Yes, it uses the `+`, but it uses `2.+` specifically. Since starting with 2.0.0 we are using semantic versioning, a given application using 2.xx shouldn't care about updates since they won't break backwards compatibility, so there should be no surprises there.

**Comment by filipedelimabrito** [comment]
Sorry, but I still think it is not the good way to go.

It's not about your promises about not breaking the changes here, it about an official recommendation.

FYI, hope it helps you understanding why: https://developer.android.com/studio/build/optimize-your-build#static_dependency_versions

If you think the info above is useful, then consider reopening this issue.
Thank you.

**Comment by saghul** [comment]
We are not forcing developers to adopt any model. We just provide an example and it's ultimately their resposibility to decide what to do.

That said, adding that link with a recommendation to the README sounds like a sensible way forward. Would you be up for submitting a PR to the documentation?

**Comment by filipedelimabrito** [comment]
I have no problems submitting a PR here, unless you don't want it. IMO it'd better to specify the version number as it should be (following the recommendation) instead of leaving it in this way with that link above.

**Comment by saghul** [comment]
I'd take a PR which recommends specifying the version and includes the link you mentioned before, but not one that set's the version to 2.0.0.

**Comment by filipedelimabrito** [comment]
If you think that it's better to keep something not recommended instead of changing the README.md after each release here do it then.

**Comment by saghul** [comment]
You keep complaining, but when I offered the solution of updating the README to say something like "this is an example, use the full version you need in your project, go read this link, etc." you just keep complaining instead of helping out. It takes less time to write that text than to keep compaining just because I don't agree with you.

I'm not going to continue replying to this thread. If you want to send a PR, great, I already explained how I think it's best. If not, that's ok too.

**Comment by filipedelimabrito** [comment]
I'm not complaining - unfortunately you're thinking wrong.

I have created this issue in order to give you an alert to avoid using `+`
on the Android SDK version number.

You closed this issue with the following comment:

> I consider the README now to be correct. Yes, it uses the +, but it uses 2.+ specifically.

Then I told you that it is still a not good way to go - I gave you an official reference that puts your comment down! But you still want to leave it in that way.

I'm sorry, but I can not agree to this and send a PR that goes against what I think and what official documentation teaches me. Sorry.

I hope you have a a good day and a happy coding.
Sorry for any disturb here.
